source/auth.h

```diff
@@ -16,6 +16,20 @@
  */
 #include "module.h"
 
+struct aws_credentials;
+
+PyObject *aws_py_credentials_new(PyObject *self, PyObject *args);
+
+/**
+ * Create Capsule to own pre-existing aws_credentials struct.
+ * The aws_credentials are destroyed when the capsule is destroyed.
```

Comment:
this seems hecka dangerous. Why are we doing this?

Reply:
oh. we're not anymore. I deleted this implementation